lib/task/import/validate/csvFieldLengthValidator.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * CSV field length test. Define fields to be checked and their max lengths.
+ * Allow to trigger either warning or error by column type.
+ *
+ * @author     Steve Breker <sbreker@artefactual.com>
+ *
+ * @internal
+ * @coversNothing
+ */
+class CsvFieldLengthValidator extends CsvBaseValidator
+{
+    const TITLE = 'Field Length Check';
+    // Add fields to check legth of here with max len. Anything larger than this value will
+    // trigger the default action.
+    protected $fieldMaxSizes = [
+        'culture' => 6,
```

Comment:
This might be a problem for Valencian users.  In our data fixtures we allow 'ca@valencia' as a valid culture value (e.g.: https://github.com/artefactual/atom/blob/qa/2.x/data/fixtures/menus.yml#L21).  The AtoM `culture` column definition is `varchar(16)` ([ref](https://github.com/artefactual/atom/blob/qa/2.x/data/sql/lib.model.schema.sql#L95)) so I would use a value of 11 or 16.

Reply:
Fixed. Good catch.

Reply:
Updated to use 11.